spec/models/recipe_spec.rb

```diff
@@ -1,5 +1,38 @@
+  expect(recipe.errors[:ready_in_minutes]).to_not be_empty
+end
+it "should have cooking_minutes" do 
+  recipe = Recipe.create title:"Gluten Free Spaghetti & Meatballs", ingredients: "1 (12 ounce) box Barilla® Gluten Free Spaghetti
+  * ½ pound ground beef 1 slice gluten free bread ⅓ cup milk Salt and black pepper to taste 1 egg yolk  ½ cup grated Parmigiano Reggiano cheese.", instructions: 'In a large bowl, combine gluten free bread and milk. Let bread soak, then squeeze out excess milk. In the same bowl, add bread, egg, ground beef, 1/2 the parsley, 1/2 the cheese, and season with salt and pepper. Roll into 3/4 inch balls &amp; set aside. In a large skillet, gently cook the garlic and chili flakes in olive oil. Add meatballs and cook thoroughly. Add the cherry tomatoes and saute for 2 minutes, then season with salt and pepper to taste. Deglaze with wine and reduce mixture by half. Cook the pasta according to package directions, then reserve 1/2 cup of the cooking liquid. Drain the pasta then toss with the sauce and add the reserved cooking liquid. Cook the pasta in the sauce for one minute, remove the skillet from the heat and fold in the remaining cheese and parsley.', ready_in_minutes: 25, diet_type: "Gluten-Free"
+  expect(recipe.errors[:cooking_minutes]).to_not be_empty
+end
+# it "should have a diet_type" do 
+#   recipe = Recipe.create title:"Gluten Free Spaghetti & Meatballs", ingredients: "1 (12 ounce) box Barilla® Gluten Free Spaghetti
+#   * ½ pound ground beef 1 slice gluten free bread ⅓ cup milk Salt and black pepper to taste 1 egg yolk  ½ cup grated Parmigiano Reggiano cheese.", instructions: 'In a large bowl, combine gluten free bread and milk. Let bread soak, then squeeze out excess milk. In the same bowl, add bread, egg, ground beef, 1/2 the parsley, 1/2 the cheese, and season with salt and pepper. Roll into 3/4 inch balls &amp; set aside. In a large skillet, gently cook the garlic and chili flakes in olive oil. Add meatballs and cook thoroughly. Add the cherry tomatoes and saute for 2 minutes, then season with salt and pepper to taste. Deglaze with wine and reduce mixture by half. Cook the pasta according to package directions, then reserve 1/2 cup of the cooking liquid. Drain the pasta then toss with the sauce and add the reserved cooking liquid. Cook the pasta in the sauce for one minute, remove the skillet from the heat and fold in the remaining cheese and parsley.', ready_in_minutes: 25,cooking_minutes: 15,  
+#   expect(recipe.errors[:diet_type]).to_not be_empty
+#   end
```

Comment:
It'd be good to uncomment this test to see if it's working

Reply:
We just uncommented it and it is not working.

Reply:
jk it worked